plugins/catalog-import/src/components/StepPrepareCreatePullRequest/StepPrepareCreatePullRequest.test.tsx

```diff
@@ -53,12 +53,17 @@ describe('<StepPrepareCreatePullRequest />', () => {
     post: jest.fn(),
   };
 
+  const configApi = {
+    getOptionalString: () => undefined,
+  };
```

Comment:
don't mock this one, use `new TestConfigApi({ /* actual config structure as a json object */ })` from `@backstage/test-utils`

Reply:
I couldn't find any method named like that, did you mean `ConfigReader` from `@backstage/core-app-api`?
So something like:
```tsx
const configApi = new ConfigReader({
    catalog: {
      filename: undefined,
    },
  });
```

Reply:
Yep that works fine too, they are basically equivalent but the other one is geared toward tests. You may be on an earlier version then of test-utils that doesn't have that yet

Reply:
Ah, re-checked the changelog for `@backstage/test-utils` and you probably meant `MockConfigApi` instead of `Test...`. I was previously only checking methods starting with `test` so missed it. Will use that instead.